dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -121,7 +120,7 @@ export interface AdProps {
 }
 
 export interface RegionalAdProps extends AdProps {
-	adRegion: AdRegion;
+	adRegion?: AdRegion;
 }
 
 export const RegionalAd = ({
```

Comment:
Given `adRegion` is now an optional prop we could change the name of the `RegionalAd` component (and `RegionalAdProps`), as at the moment it implies it probably always has a region? `Ad` would probably suffice but it'd require the existing `Ad` component be named something more specific?

Reply:
Snap :) @zekehuntergreen and I discussed along similar lines.

We could swap the hierarchy so there is now a base 'Ad' Component which the `RegionalAd` will loop over, whereas `StickyAd` will call the base `Ad` directly.

Reply:
I like that!